app/controllers/api/v1/trails_controller.rb

```diff
@@ -2,11 +2,13 @@ class Api::V1::TrailsController < ApiController
 
     def index
         trails = Trail.all
+        
         render json: trails
     end
     
     def show
         trail = Trail.find(params[:id])
+
         render json: trail
```

Comment:
so, you are going to need to rework this a smidge by adding a TrailSerializer

that serializer should link reviews as well (see clinic), which will also mean you will want a ReviewSerializer

Once you have those, you wont need two fetch requests (one for trail, one for reviews) but instead one fetch request that can do both! faster and simpler